TestAssets/TestProjects/PerformanceTestProjects/TwoTargetGraphLarge/TwoTargetLargeP6/project.json

```diff
@@ -1,13 +1,19 @@
 {
   "version": "1.0.0-*",
-  "dependencies": {
-    "Microsoft.NETCore.App": {
-      "type": "platform",
-      "version": "1.0.0-rc3-004324"
-    }
-  },
+  "dependencies": { },
   "frameworks": {
-    "netcoreapp1.0": {},
-    "netstandard1.5": {}
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc3-004336"
+        }
+      }
+    },
+    "netstandard1.5": {
```

Comment:
No idea. @pakrym added these tests, maybe he can comment.

From the name of the projects and the tests, they are trying to target 2 TFMs at the same time. I debated on switching it to `net4xy`, but figured this would be wrong since it wouldn't work cross-plat.